README.md

```diff
@@ -87,10 +87,20 @@ npm start
   ]
 ]
 ```
+
+* `CONNECTOR_NOTIFICATION_VERIFY` (default: `'true'`) The connector verifies the signatures on notifications.
```

Comment:
That's why we default to `true`. Is there a reason to make this non-configurable?